### PR TITLE
Updates breadcrumbs and your requirements page content

### PR DESCRIPTION
**Pull request by DilwoarH**
Ticket: https://trello.com/c/FiR3NLyZ/679-buyer-account-page

# Requirements Page
## Before
screen shot 2017-09-12 at 15 16 24

## After
screen shot 2017-09-12 at 15 16 17

# Breadcrum
## Before
breadcrumbs-before

## After
breadcrumbs-after

### CODE REVIEW

**Comment by samuelhwilliams** [comment]
Have created a PR to update breadcrumb content in the buyer-frontend app as well. https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/607

File: app/templates/buyers/brief_overview.html [hunk 1]
```diff
@@ -14,6 +14,13 @@
...
         "link": url_for(".buyer_dashboard"),
         "label": "Your account"
+      },
+      {
+        "link": url_for(".buyer_dos_requirements"),
+        "label": "Your requirements"
+      },
+      {
+        "label": brief.title
       }
```
Comment (DilwoarH) [inline app/templates/buyers/brief_overview.html]: remove

File: app/templates/buyers/dashboard.html [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
         "link": "/",
         "label": "Digital Marketplace"
+    },
+    {
+        "link": url_for('buyers.buyer_dashboard'),
+        "label": "Your account"
+    },
+    {
+        "label": "Your requirements"
     }
```
Comment (DilwoarH) [inline app/templates/buyers/dashboard.html]: remove

File: app/templates/buyers/brief_overview.html [hunk 3]
```diff
@@ -14,6 +14,13 @@
       {
         "link": url_for(".buyer_dashboard"),
         "label": "Your account"
+      },
+      {
+        "link": url_for(".buyer_dos_requirements"),
+        "label": "Your requirements"
+      },
+      {
+        "label": brief.title
```
Comment (samuelhwilliams) [inline app/templates/buyers/brief_overview.html]: I don't think we should add `brief.title` unless it's been specifically asked for, it's slightly inconsistent for that flow. Spoken to @idavidmcdonald and we'll leave it out for now, and maybe revisit breadcrumbs consistency as another piece of work later.

File: app/templates/buyers/index.html [hunk 4]
```diff
@@ -16,19 +16,19 @@
...
         context = current_user.email_address,
         heading = current_user.name
     %}
     {% include 'toolkit/page-heading.html' %}
     {% endwith %}
   </div>
-  <div class="column-full dm-account-overview-options">
+  <div class="column-two-thirds dm-account-overview-options">
     
     <div class="framework">
```
Comment (samuelhwilliams) [inline app/templates/buyers/index.html]: We should still be able to do this without a specific framework div as there isn't really anything special about the content here.

Comment (samuelhwilliams) [reply app/templates/buyers/index.html]: Yeah we can get the styling we want without using extra divs by just having everything in a single dmspeak div.

Comment (samuelhwilliams) [reply app/templates/buyers/index.html]: If we've added SCSS somewhere to support the frameworks div, we should remove that.

**Comment by samuelhwilliams** [comment]
Also as David touches upon, some of these templates will need to be feature flagged so that they retain their current (i.e. old) breadcrumbs until we activate direct award later.

File: app/templates/buyers/index.html (hunk 4 above)
Comment (DilwoarH) [reply app/templates/buyers/index.html]: ok sure I'll get those changes in

**Comment by DilwoarH** [comment]
requested changes implemented and additional CSS files which are no longer required are removed.

**Comment by DilwoarH** [comment]
@samuelhwilliams - all your requested changes have been made now

**Comment by samuelhwilliams** [comment]
@DilwoarH Cool, I will look it over again now. In the mean time can you squash these commits down please - we have some commits adding a feature and removing it again in the same PR, which is not good.

**Comment by samuelhwilliams** [comment]
With the feature turned off, there are still some bad breadcrumbs. On /buyers dashboard, you have a link to 'Your account' which goes to the current page (i.e. breadcrumb shouldn't be there). On requirements questions, you have a link to 'Your account' and 'Your requirements' which are visually the same page, but on different URLs (i.e. you're exposing the new briefs dashboard URL). See examples below.

screen shot 2017-09-14 at 09 41 49

screen shot 2017-09-14 at 09 41 26

**Comment by DilwoarH** [comment]
How did you remove the feature flag? changing the date does not work, you have to remove the whole flag from config.

screen shot 2017-09-14 at 10 06 04

**Comment by samuelhwilliams** [comment]
I set the flag to False. The page you've posted isn't the same one I have. If you go into one of the questions (e.g. `Specialist Role`) you should see what I posted.

**Comment by idavidmcdonald** [comment]
Some good spots Sam. Let me know when this is ready after a bunch of changes are made and I'll do another quick review too as things have changed a fair amount since my previous approving review.

I might also suggest you highlight the breadcrumbs in particular to be tested by Andrew in QA as they could be easy to miss.

**Comment by DilwoarH** [comment]
Good spot @samuelhwilliams - it's been updated.

**Comment by DilwoarH** [comment]
Breadcrumbs updated for Dashboard

screen shot 2017-09-14 at 11 49 36

**Comment by idavidmcdonald** [comment]
If we are feature flagging the breadcrumbs, we also need to feature flag the title heading on the buyer account page that says the buyer name etc and also the title of the page changing from 'Your account' to 'Your requirements'.

By developing on preview it's ending up feeling that it's a lot of hassle to feature flag every change you are making...

One easier solution would be to release the new homepage structure now, saving us issues with logic around feature flags, before the actual functionality to save searches is launched

**Comment by DilwoarH** [comment]
@samuelhwilliams @idavidmcdonald - is this good to merge?

**Pull request merged**